pytest.ini

```diff
@@ -15,3 +15,6 @@ python_classes =
 python_functions =
     test_
     it_
+
+env =
+    XDG_RUNTIME_DIR=
```

Comment:
The blanking of XDG_RUNTIME_DIR inside conftest isn't enough, because pytest could import the module before the fixture runs. (I also tried reload, gross though it may be, and for #reasons it didn't work.)